CI/flatpak/com.obsproject.Studio.json

```diff
@@ -403,6 +440,11 @@
     {
       "name": "obs",
       "buildsystem": "cmake-ninja",
+      "build-options": {
+        "env" : [
+          "CARGO_HOME=/run/build/obs/cargo"
+        ]
```

Comment:
Same as first conversation.

Reply:
Converted from array to object in https://github.com/obsproject/obs-studio/pull/7926/commits/591189c171d946249eeda3c6453a042434bdac47